scripts/o3de/o3de/manifest.py

```diff
@@ -314,6 +314,24 @@ def get_project_external_subdirectories(project_path: pathlib.Path) -> list:
                         project_object['external_subdirectories'])) if 'external_subdirectories' in project_object else []
     return []
 
+def get_project_engine(project_path: str or pathlib.Path) -> pathlib.Path or None:
+    project_object = get_project_json_data(project_path=project_path)
+    if project_object:
+        engine_name = project_object.get('engine','')
```

Comment:
nitpick: missing space after the comma in the get call.
```suggestion
        engine_name = project_object.get('engine', '')
```